visualization/app/codeCharta/ui/sortingOptionDialog/sortingOptionDialog.component.ts

```diff
@@ -0,0 +1,40 @@
+import "./sortingOptionDialog.component.scss"
```

Comment:
As its not a pop-up dialog like the others (which use the $mdDialog component from angular), it's definitely a separate component. I would therefore remove the name dialog here, as it could become confusing. How about just `sortingOption.component` @RomanenkoVladimir, @NearW? :)

@RomanenkoVladimir unfortunately renaming a component is not as easy as creating one 😄 some manual refactoring is necessary.. if you have questions, always feel free to ask 🎉